File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/kontonummer/UtenlandskKontoInformasjon.java
package no.nav.personopplysninger.features.endreopplysninger.domain.kontonummer;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class UtenlandskKontoInformasjon {
    private Bank bank;
    private String landkode;
    private String swift;
    private String valuta;

    public Bank getBank() {
        return bank;
    }

    public String getLandkode() {
        return landkode;
    }

    public String getSwift() {
        return swift;
    }

    public String getValuta() {
        return valuta;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysningergammel/domain/kontonummer/UtenlandskKontoInformasjon.kt
package no.nav.personopplysninger.features.endreopplysningergammel.domain.kontonummer

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class UtenlandskKontoInformasjon {
    val bank: Bank? = null
    val landkode: String? = null
    val swift: String? = null
    val valuta: String? = null
}
